### PR TITLE
Updates to overhaul_readme work based on feedback

### PR DESCRIPTION
**Pull request by apollokit**
Updates to overhaul_readme work based on feedback

### CODE REVIEW

**Comment by ericjunkins** [comment]
@apollokit Did you need any additional feedback or help on this?

File: setup/rpi.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ First we'll create a workspace for the rover code.
 mkdir -p ~/osr_ws/src && cd ~/osr_ws
 
 # Source your newly created ROS environment
-source /opt/ros/$ROS_DISTRO/setup.bash
+source /opt/ros/melodic/setup.bash
```
Comment (Achllle) [inline setup/rpi.md:99]: Don't see why this is necessary. `ROS_DISTRO` is a real variable set when you install ROS, so this is not a placeholder that the user needs to edit.

Comment (JHPHELAN) [reply setup/rpi.md:99]: In section "4.1 Setup ROS build environment", at the step "Source your newly created ROS environment", entering the command 
ubuntu@ubuntu:~/osr_ws$ source /opt/ros/$ROS_DISTRO/setup.bash
produces the error message:
"-bash: /opt/ros//setup.bash: No such file or directory"
Replacing the variable "$ROS_DISTRO" with "melodic": 
ubuntu@ubuntu:~/osr_ws$ source /opt/ros/melodic/setup.bash
eliminates the error:

**Comment by apollokit** [comment]
@ericjunkins ha, not sure why I marked it as a draft. I think it's ready for a merge, given at least Achille has reviewed the only file changed.

**Pull request merged**